indigo_app/forms.py

```diff
@@ -120,10 +120,13 @@ def clean_options(self):
 class TaskForm(forms.ModelForm):
     class Meta:
         model = Task
-        fields = ('title', 'description', 'work', 'document', 'labels', 'workflows')
+        fields = ('title', 'description', 'work', 'document', 'labels', 'workflows', 'assigned_to')
 
-    labels = forms.ModelMultipleChoiceField(queryset=TaskLabel.objects, widget=forms.CheckboxSelectMultiple, required=False)
-    workflows = forms.ModelMultipleChoiceField(queryset=Workflow.objects, widget=forms.CheckboxSelectMultiple, required=False)
+    labels = forms.ModelMultipleChoiceField(queryset=TaskLabel.objects, widget=forms.CheckboxSelectMultiple,
+                                            required=False)
+    workflows = forms.ModelMultipleChoiceField(queryset=Workflow.objects, widget=forms.CheckboxSelectMultiple,
+                                               required=False)
+    assigned_to = forms.ModelChoiceField(queryset=User.objects, empty_label='Unassigned', required=False)
```

Comment:
Do we want to scope this right now to only the users that have the task's country badge? They're the only ones who can make changes to things in this country. We'd need to do that in the form's constructor. I think we track the country permissions via the Editor model.